plugins/hydra_joblib_launcher/tests/test_joblib_launcher.py

```diff
@@ -67,3 +70,34 @@ def test_example_app(hydra_sweep_runner: TSweepRunner) -> None:
         assert sweep.returns is not None and len(sweep.returns[0]) == 4
         for ret in sweep.returns[0]:
             assert tuple(ret.overrides) in overrides
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "overrides, expected",
+    [
+        ("hydra.launcher.batch_size=1", ("hydra.launcher.batch_size", "1")),
```

Comment:
Why put the two values in a tuple?
```
"overrides, tested_key,expected_value",
```

Also, I am not sure testing the key and value here adds any value. this is testing Hydra's overriding capability, which is very well tested already.